modules/client-configuration/pages/package-management.adoc

```diff
@@ -47,3 +47,51 @@ When the profile has been created, you can use it to install more clients with t
 . In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
     To compare with a stored profile, select the profile and click btn:[Compare].
 . To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.
+
+
+
+== Orphaned Packages
+
+Orphaned packages do not have associated channels.
+Therefore, orphaned packages are usually not available to {productname} clients and cannot install such packages.
+
+Orphaned packages can exist as a result of one of the following events:
```

Comment:
Following my first suggestion, I'd reinforce here that we're not talking about a new term, but about a state of a package in uyuni/suma.

```suggestion
A package can become orphaned as a result of one of the following events:
```